pkg/loader/load.go

```diff
@@ -438,8 +451,39 @@ func (s *loaderImpl) execDMLs(dmls []*DML) error {
 	return errors.Trace(err)
 }
 
+func (s *loaderImpl) createMarkTableDDL() error {
+	markTableDataBase := loopbacksync.MarkTableName[:strings.Index(loopbacksync.MarkTableName, ".")]
+	createDatabaseSQL := fmt.Sprintf("create database IF NOT EXISTS %s;", markTableDataBase)
+	createDatabase := DDL{SQL: createDatabaseSQL}
+	if s.db == nil || s.loopBackSyncInfo == nil {
+		log.Error("loaderImpl db point object is nil,maybe not create mark table and database or not need creat")
```

Comment:
`s.db` should never be nil.
Do not log an error msg and return nil if `s.loopBackSyncInfo` is nil cause seems it's valid to be nil as default value you set it.

make clear when we need to create the table, if we don't need, should there's no ans error msg.
if we do need, should success to create it or return error and `Loader` failed to work.